packages/ui-avatar/src/Avatar/index.tsx

```diff
@@ -135,7 +144,7 @@ class Avatar extends Component<AvatarProps, AvatarState> {
           onLoad={this.handleImageLoaded}
           aria-hidden="true"
         />
-        {!this.state.loaded && this.renderInitials()}
+        {!this.state.loaded && this.renderContent()}
```

Comment:
A potential issue here: If I set the src prop, it will cause loaded set to `true` when the image loads. Then if I set the `src` to `undefined` to display the initials/icon it will not show anything because loaded is never set to `false`

Reply:
It has been like this from the start, so not a new issue (not very likely problem either), but added the logic for it, just in case. Thanks!